templates/arista_eos_show_ip_bgp_summary.template

```diff
@@ -6,12 +6,15 @@ Value MSG_RCVD (\d+)
 Value MSG_SENT (\d+)
 Value IN_QUEUE (\d+)
 Value OUT_QUEUE (\d+)
-Value UP_DOWN (\d+:\d+:\d+|\d+)
+Value UP_DOWN (\d+\S\d+\S|\d+:\d+:\d+)
```

Comment:
seems too complex, what happens when under an hour or under a minute? I'm ok with being looser on the capture groups and if the whole line is accounted for.